Makefile

```diff
@@ -0,0 +1,20 @@
+golangci-lint-version = v1.56.1
+ifeq ($(GOOS),windows)
+  WORKSPACE := ${shell go env GOPATH}
+else
+  WORKSPACE := ${shell go env GOPATH | rev | cut -d: -f1 | rev}
+endif
+
+.PHONY: install-golangci-lint
+install-golangci-lint:
+	$(call print-target)
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh >lint-install.sh
+	chmod u+x ./lint-install.sh && ./lint-install.sh -b $(WORKSPACE)/bin $(golangci-lint-version)
+	$(RM) ./lint-install.sh
```

Comment:
This could be moved to a separate script in order to:
1. Check if the correct version of golangci-lint is already installed, skipping if so.
2. Give us more ways to avoid a temporary `lint-install.sh` file, if that's something we want to do. For example, if we're just trying to avoid a line-length issue, each command could be wrapped in a function, and then the actual command could look like `download | execute`

Reply:
Yeah, that would be nice to have but maybe let's do it in separate PRs.